SU-SWS/stanford_news
# EARTH-159 Changed hero banner to top media and added feature image field

# READY FOR REVIEW

# Summary
- Removed hero banner in favor of top_media field to use hero banner or video
- configured panelizer with new field.
- new featured image field not displayed on node.

# Needed By (Date)
- End of sprint

# Steps to Test

1. Checkout branch and same branch in [stanford_layouts](https://github.com/SU-SWS/stanford_layouts/pull/4) & [SE3_BLT](https://github.com/stanford-earth/SE3_BLT/pull/43)
2. cache clear then Config Import, features import all
3. create a news content
4. populate the hero banner or the video
5. verify it looks correct and the page title displays if video used or hero banner used with nothing in the "super heading" field (with variant set to "page title").

# Associated Issues and/or People
- https://github.com/stanford-earth/SE3_BLT/pull/43
- https://github.com/SU-SWS/stanford_layouts/pull/4

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
